src/DepartmentService.Business/Department/CreateDepartmentCommand.cs

```diff
@@ -64,7 +64,7 @@ public class CreateDepartmentCommand : ICreateDepartmentCommand
 
       #region Deactivated previous department user records
 
-      if (request.Users != null)
+      if (request.Users.Any())
```

Comment:
request.Users?.Any() maybe?

Reply:
VS don't like this variation. There appear a problem with converting "bool?" to "bool".
So, as I understand, it's possible to stay variation with "request.Users.Any()" or change all if-construction.

Reply:
Check for null is in validation. So, it can't be null here.